elasticsearch_async/connection.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
             connector=aiohttp.TCPConnector(
                 loop=self.loop,
                 verify_ssl=verify_certs,
-                conn_timeout=self.timeout,
+                keepalive_timeout=self.timeout,
```

Comment:
I don't think we want to change the `keepalive_timeout` - we should leave the `conn_timeout`